scripts/upload.gradle

```diff
@@ -35,8 +35,21 @@ uploadArchives {
             pom.version = rootProject.ext.Version
 
             pom.project {
-                description 'git rev-parse HEAD'.execute([], projectDir).text.trim()
-                // 最后一次git提交的hash
+                name 'APNG4Android'
+                description 'Android animation support for APNG & Animated WebP & Gif'
+                url 'https://github.com/penfeizhou/APNG4Android'
+                licenses {
+                    license {
+                        name = 'The Apache License, Version 2.0'
+                        url = 'http://www.apache.org/licenses/LICENSE-2.0.txt'
+                    }
+                }
+                developers {
+                    developer {
+                        id = 'penfeizhou'
+                        name = 'osborn'
```

Comment:
This is just copied from your Github profile, so please update as appropriate. I wasn't sure if other developers (or how many) should be added here either, but please also update that as appropriate.